docs/src/pages/homepage.module.css

```diff
@@ -0,0 +1,275 @@
+  font-weight: 700;
+  font-size: 24px;
+}
+
+.tagline2 {
+  margin: 0;
+  margin-bottom: 1.5em;
+  font-weight: 400;
+  font-size: 24px;
+}
+
+.getStartedLink {
+  display: inline-flex;
+  background-color: var(--ifm-color-primary-dark);
+  color: var(--ifm-color-white);
+  border-radius: 4px;
+  padding: 0.8em 2em;
+  font-weight: 700;
+  align-items: baseline;
+  text-decoration: none;
+}
+
+.getStartedLink:hover {
+  background-color: var(--ifm-color-primary);
+  color: #fff;
+  text-decoration: none;
+}
+
+/*
+  Why choose Nebari?
+*/
+
+.pitchSection {
+  padding: 75px;
+}
+
+.pitchList {
+  display: flex;
+  flex-flow: row wrap;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  justify-content: center;
+}
+
+.pitchItem {
+  display: flex;
+  flex-direction: column;
+  flex: 0 1 auto;
+  width: 33.33%;
+  text-align: center;
+  align-items: center;
+  font-size: 19px;
+  line-height: 26px;
+  padding: 35px;
+}
+
+@media screen and (max-width: 996px) {
+  .pitchItem {
+    flex: 0 1 543px;
+  }
+}
+
+.pitchItemHeading {
+  font-size: var(--ifm-h4-font-size);
+  margin-bottom: 1em;
+  font-weight: 600;
+}
+
+/*
+  Deploy anywhere
+*/
+
+.platformsSection {
+  background-color: var(--ifm-color-white);
+  color: var(--ifm-color-black);
+}
+
+.platformsSectionContent {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.platformsHeading {
+  text-align: left;
+  margin-top: 0.5em;
+  margin-bottom: 1.5em;
+}
+
+.platformsLeft {
+  width: 50%;
+  padding: 75px;
+}
+
+/* This is to put a max-width on the text-column so it stays about the same
+height as the column of logos to the right */
+.platformsTextColumn {
+  max-width: 540px;
+  font-size: 19px;
+  line-height: 36px;
+}
+
+.platformsRight {
+  width: 50%;
+  padding: 75px;
+}
+
+.platformsList {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.platformsItem {
+  display: flex;
+  background-color: #fff;
+  box-shadow: 0px 3px 15px rgba(0, 0, 0, 0.1);
+  border-radius: 25px;
+  height: 88px;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 30px;
+  flex: 1 1 397px;
+  width: 397px;
+  max-width: 397px;
+  min-width: 292px;
+}
+
+.platformsItem:nth-child(2n) {
+  position: relative;
+  left: 59px;
+}
+
+@media screen and (max-width: 996px) {
+  .platformsLeft {
+    width: 100%;
+    padding-bottom: 0;
+  }
+  .platformsRight {
+    width: 100%;
+  }
+  .platformsList {
+    justify-content: flex-end;
+  }
+}
+
+/*
+  Get started
+*/
+
+.getStartedSection {
+  background-color: var(--ifm-color-black);
+  color: var(--ifm-color-white);
+  padding: 82px;
+  text-align: center;
+}
+
+.getStartedLink2 {
+  display: inline-flex;
+  align-items: baseline;
+  color: var(--ifm-color-white);
+  font-weight: 400;
+  font-size: 19px;
+  line-height: 36px;
+  text-decoration: none;
+  text-align: center;
+  border-bottom: transparent 1px solid;
+  margin: 2em 0;
+}
+.getStartedLink2:hover {
+  text-decoration: none;
+  color: #fff;
```

Comment:
is there a particular reason why you swap colour from `--ifm-color-white` to `#fff`?
I think the colour difference is barely there 👀  

For the underline you can use `--nebari-links-dark`

Reply:
I'll look into it.